lib/ansible/modules/slurp.py

```diff
@@ -22,6 +22,14 @@
     type: path
     required: true
     aliases: [ path ]
+  armor:
+    description:
+     - To safely deliver the data requested, armor it by base64 encoding it.
```

Comment:
Should add a point saying this is not supported on Windows (yet). Will have to add it separately to the `ansible.windows` collection once this has been merged.

Reply:
good point, totally forgot windows had slurp